src/main/java/oshi/software/os/mac/local/CentralProcessor.java

```diff
@@ -32,8 +37,14 @@
 	 * @return Processor vendor.
 	 */
 	public String getVendor() {
-		if (_vendor==null)
-			_vendor=ExecutingCommand.getFirstAnswer("sysctl -n machdep.cpu.vendor");
+		if (_vendor == null) {
+			int[] mib = { SystemB.CTL_MACHDEP, SystemB.MACHDEP_CPU,
+					SystemB.MACHDEP_CPU_VENDOR };
+			IntByReference size = new IntByReference(100);
+			Pointer p = new Memory(size.getValue());
+			SystemB.INSTANCE.sysctl(mib, mib.length, p, size, null, 0);

```

Comment:
All the API calls we're using return an integer, 0 if successful and 1 if not.  I will put in checks to provide sensible defaults.